### DISCUSSION TITLE
composer.json suggest  Magento-Two-factor-Authentication

### DISCUSSION THREAD

**Discussion by kkrieger85** [OpenMage/magento-lts#2153] [Ideas]
In #307, some agreed to use better encryption and security. 

Therefore we should add magento-hackathon/Magento-Two-factor-Authentication as suggestion to composer.json

**Comment by sreichel** [comment]
+1

**Comment by colinmollenhour** [comment]
Rather than adding suggestions to core what about adding a sort of "starter" repo that sets up an OpenMage LTS project with a handful of no-brainer extensions already added (such as 2fa). This would serve as a reference for how to create a new (or migrate to) an OpenMage LTS project. Personally I still don't know the best or most proper way to do this as I'm stuck in my old ways.. :)

Any volunteers to create the "starter" project?

**Comment by kkrieger85** [comment]
@colinmollenhour What about both? 
1) adding these modules as composer suggestion AND
2) creating a starter project

**Comment by kkrieger85** [comment]
For "starter" I use https://github.com/kkrieger85/coronamage/blob/master/composer.json 

Maybe, this could be a good point to start a "starter project". 

But: This is not tested with composer2 
Also: This is not tested with newer PHP versions
